Source/Core/DolphinWX/FifoPlayerDlg.cpp

```diff
@@ -80,197 +80,197 @@ void FifoPlayerDlg::CreateGUIControls()
 	m_Notebook = new wxNotebook(this, wxID_ANY);
 
 	{
-	m_PlayPage = new wxPanel(m_Notebook, wxID_ANY, wxDefaultPosition, wxDefaultSize, wxTAB_TRAVERSAL);
-	wxBoxSizer* sPlayPage;
-	sPlayPage = new wxBoxSizer(wxVERTICAL);
+		m_PlayPage = new wxPanel(m_Notebook, wxID_ANY, wxDefaultPosition, wxDefaultSize, wxTAB_TRAVERSAL);
```

Comment:
Thats somewhat inconsistent with what we do with namespaces. Theres a decent reason for not doing it there, and I guess you could apply the same reasoning here (since its only scope related, and not an actual loop/branch/whatever).